src_c/font.c

```diff
@@ -425,9 +483,10 @@ font_render(PyObject *self, PyObject *args)
     Uint8 rgba[] = {0, 0, 0, 0};
     SDL_Surface *surf;
     const char *astring = "";
+    int wraplength = 0;
```

Comment:
We should add an error handler for negatives. No segmentation fault happens if we do, but we get this error: `pygame.error: Invalid parameter 'wrapLength'` which I think is ugly.